app/helpers/mustache_helper.rb

```diff
@@ -281,6 +342,74 @@ def content
     }
   end
 
+  def settings
```

Comment:
Assignment Branch Condition size for settings is too high. [20/15]<br>Method has too many lines. [64/10]